cmd/bblfsh-performance/endtoend/endtoend.go

```diff
@@ -22,14 +22,20 @@ import (
 	"gopkg.in/src-d/go-log.v1"
 )
 
-const prefix = "bench_"
+const (
+	// TODO(lwsanty): flag
+	prefix = "bench_"
```

Comment:
Can you please also document what this is a prefix for? I realize it's unexported, but `prefix` is a fairly generic name.

Reply:
Done

Reply:
Makes sense. Thanks for adding the comment, that helps. Would you be willing to also give it a more self-describing variable name? Maybe, say `fileFilterPrefix` or something like that?

Reply:
Done